examples/sock_tcp_echo/Makefile

```diff
@@ -0,0 +1,40 @@
+# name of your application
+APPLICATION = sock_tcp_echo
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# default to using GNRC
+LWIP ?= 0
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+USEMODULE += sock_tcp
+USEMODULE += netdev_default
+
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+USEMODULE += netutils
+
+ifeq (1, $(LWIP))
+  USEMODULE += ipv6_addr
+  USEMODULE += lwip_ipv6_autoconfig
+  USEMODULE += lwip_netdev
+else
+  USEMODULE += auto_init_gnrc_netif
+  USEMODULE += gnrc_ipv6_default
+  KCONFIG_ADD_CONFIG += app.gnrc.config
```

Comment:
so something with this line isn't working

adding a `CFLAGS=-DCONFIG_GNRC_TCP_RCV_BUFFERS=2` as @brummer-simon suggested works

Reply:
That's the way I had it [initially](https://github.com/RIOT-OS/RIOT/pull/16739#discussion_r712793668), I revert back to it.